relay/nakama/utils/testutils.go

```diff
@@ -1,4 +1,4 @@
-package main
+package utils
```

Comment:
Per our conversation, this util package should just have utility functions for the production code. Test helpers should just live in the package where they are used OR put in a "testutils" package (if multiple packages will make use of them).